### PR TITLE
Add Symfony 4 support

### PR DESCRIPTION
**Pull request by pablothedude**
Add Symfony 4 support

### CODE REVIEW

File: src/Value/HealthReport.php [hunk 1]
```diff
@@ -77,7 +77,9 @@ public function getStatusCode()
 
     public function jsonSerialize()
     {
-        $report['status'] = $this->status;
+        $report = [
+            'status' => $this->status
+        ];
```
Comment (MKodde) [inline src/Value/HealthReport.php:82]: Was this an MD fail?

File: composer.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
   "type": "symfony-bundle",
-  "description": "A Symfony 3 bundle that facilitates health and info endpoints to a Symfony application. The bundle is backwards compatible with Symfony 2 projects.",
+  "description": "A Symfony 4 bundle that facilitates health and info endpoints to a Symfony application. The bundle is backwards compatible with Symfony 2 projects.",
   "keywords": ["SURFnet", "StepUp", "OpenConext", "monitoring", "health"],
   "license": "Apache-2.0",
   "minimum-stability": "stable",
   "require": {
     "php": ">=5.4,<8.0-dev",
-    "symfony/dependency-injection": ">=2.7,<4",
-    "symfony/framework-bundle": ">=2.7,<4",
```
Comment (MKodde) [inline composer.json:11]: Aside from wanting to drop SF3 support, do these changes really break SF3 support? They seem very 3 compatible. If this is the case, we might get away with a non-BC breaking change here by leaving the SF3 support in place?

**Review by MKodde — APPROVED** [review]
Seems like a sweet and simple SF4 upgrade. Looks good to me, see some thoughts below. 

It might also be a good idea to enable PHP 72 builds on Travis, maybe disabling PHP 70 and 71.

File: src/Value/HealthReport.php (hunk 1 above)
Comment (pablothedude) [reply src/Value/HealthReport.php:82]: Yes, `$report` wasn't initialized when `$report['status']` was set.

File: composer.json (hunk 2 above)
Comment (pablothedude) [reply composer.json:11]: I removed the support for Symfony prior to 3.4. This isn't currently supported by Symfony either.

**Comment by pablothedude** [comment]
I've changed the test matrix to test 7.2 and 7.4 only.

**Pull request merged**